.gitignore

```
@@ -0,0 +1,3 @@
+/node_modules
+/.env
```

Em sửa chỗ này thành
```suggestion
.env
```
Xong chạy lệnh dưới để xóa file `.env` khỏi git. Lúc này file .env chỉ còn ở local và bỏ qua bởi git nhé. Mục đích là k đưa các thông tin bảo mật lên public
`git rm --cached example.txt`

---

đã chỉnh ạ